eventmesh-metrics-plugin/eventmesh-metrics-api/src/main/java/org/apache/eventmesh/metrics/api/model/HttpSummaryMetrics.java

```diff
@@ -120,20 +120,20 @@ public float avgHTTPBodyDecodeCost() {
         return (httpDecodeNum.longValue() == 0L) ? 0f : httpDecodeTimeCost / httpDecodeNum.longValue();
     }
 
-    private AtomicLong sendBatchMsgNumPerSecond = new AtomicLong(0);
+    private final AtomicLong sendBatchMsgNumPerSecond = new AtomicLong(0);
 
-    private AtomicLong sendBatchMsgNumSum = new AtomicLong(0);
+    private final AtomicLong sendBatchMsgNumSum = new AtomicLong(0);
 
-    private AtomicLong sendBatchMsgFailNumSum = new AtomicLong(0);
+    private final AtomicLong sendBatchMsgFailNumSum = new AtomicLong(0);
 
     // This is a cumulative value
-    private AtomicLong sendBatchMsgDiscardNumSum = new AtomicLong(0);
+    private final AtomicLong sendBatchMsgDiscardNumSum = new AtomicLong(0);
 
     public void recordSendBatchMsgDiscard(long delta) {
         sendBatchMsgDiscardNumSum.addAndGet(delta);
     }
 
```

Comment:
Adjust the position of variables, move them to the beginning of the file class, and place the methods below the variables.